src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -132,6 +131,16 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
       });
   };
 
+  private getGatewaysAsList(): string[] {
+    let gatewayList: string[] = [];
+
+    this.state.gateways.forEach(gateway => {
+      gatewayList.push(gateway.metadata.namespace + '/' + gateway.metadata.name);
+    });
+
+    return gatewayList;
```

Comment:
Also, you probably can simplify with `map`:

```suggestion
    return this.state.gateways.map(gateway => gateway.metadata.namespace + '/' + gateway.metadata.name);
```

That `map` function is magical :-)

Reply:
Thans for your review @israel-hdez! sure, good simplifications, totally agree!